docs/modelserving/observability/prometheus_metrics.md

```diff
@@ -0,0 +1,62 @@
+# Prometheus Metrics 
+
+## Exposing Prometheus Port
+
+All supported serving runtimes support exporting prometheus metrics on a specified port in the inference service's pod. The appropriate port for the model server is defined in the [kserve/config/runtimes](https://github.com/kserve/kserve/tree/master/config/runtimes) YAML files. For example, torchserve defines its prometheus port as `8082` in `kserve-torchserve.yaml`. 
+
+```yaml
+metadata:
+  name: kserve-torchserve
+spec:
+  annotations:
+    prometheus.kserve.io/port: '8082'
+    prometheus.kserve.io/path: "/metrics"
+```
+
+If needed, this value can be overridden in the InferenceService YAML. 
+
+To enable prometheus metrics, add the annotation `serving.kserve.io/enable-prometheus-scraping` to the InferenceService YAML. 
```

Comment:
There is also a global config to turn on the scraping without adding this to each inference service yaml right ?

Reply:
the info for that is in the docs linked below I can add another link here

Reply:
we expect user to patch the config map themselves right?